backend/app/pydantic_models/question.py

```
@@ -0,0 +1,7 @@
+from pydantic import BaseModel
+from typing import List
+
+class Question(BaseModel):
+    question: str
+    answer: str
+    tags: List[str] = [] 
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)